ui/StatusQ/src/systemutilsinternal.cpp

```diff
@@ -164,4 +168,33 @@ Qt::MouseButtons SystemUtilsInternal::mouseButtons()
     return QGuiApplication::mouseButtons();
 }
 
+void SystemUtilsInternal::setAndroidStatusBarIconColor(bool lightIcons)
+{
+#ifdef Q_OS_ANDROID
+    QJniObject activity = QNativeInterface::QAndroidApplication::context();
+    if (activity.isValid()) {
+        QJniObject::callStaticMethod<void>(
+            "im/status/app/StatusBarUtil",
+            "setStatusBarIconColor",
+            "(Landroid/app/Activity;Z)V",
+            activity.object<jobject>(),
+            lightIcons
+        );
+    }
+#else
+    Q_UNUSED(lightIcons);
+#endif
```

Comment:
kept the StatusQ API clean, no matter the platform. In theory there's no need to check if we're on Android to call these methods.